src/Vlingo.Common/Vlingo.Common.csproj

```diff
@@ -5,7 +5,7 @@
     
     <!-- NuGet Metadata -->
     <IsPackable>true</IsPackable>
-    <PackageVersion>0.3.0</PackageVersion>
+    <PackageVersion>0.3.1</PackageVersion>
```

Comment:
As this is a breaking change it should be rather 0.4, if we align to SemVer

Reply:
Good point. Will do.